neo4j-cli/aura/internal/subcommands/deployment/database/database.go

```diff
@@ -0,0 +1,17 @@
+package database
```

Comment:
I'm a bit confused about the `server` command, probably because I'm not that familiar with fleet management. But I see there are 2 endpoints:
* `/servers`
* `databases/../servers`

I'm trying to understand the different between these 2 (however, the description in the [api spec](https://neo4j.com/docs/aura/platform/api/specification/?urls.primaryName=Aura%20v2beta1#/deployments/get-deployment-servers) is not too helpful as the second one seems to have the same description as just `/servers`

What is the difference between these? Is there a chance of user confusion between these commands?